test/powershell/engine/Remoting/InvokeCommandRemoteDebug.Tests.ps1

```diff
@@ -0,0 +1,205 @@
+﻿##
+## PowerShell Invoke-Command -RemoteDebug Tests
+##
+
+if ($IsWindows)
+{
+    if (-not (Get-Module TestRemoting -ErrorAction SilentlyContinue))
+    {
+        $remotingModule = Join-Path $PSScriptRoot "../Common/TestRemoting.psm1"
+        Import-Module $remotingModule
+    }
+
+    $typeDef = @'
+    using System;
+    using System.Globalization;
+    using System.Management.Automation;
+    using System.Management.Automation.Runspaces;
+    using System.Management.Automation.Host;
+
+    namespace TestRunner
+    {
+        public class DummyHost : PSHost, IHostSupportsInteractiveSession
+        {
+            public Runspace _runspace;
+            private Guid _instanceId = Guid.NewGuid();
+
+            public override CultureInfo CurrentCulture
+            {
+                get { return CultureInfo.CurrentCulture; }
+            }
+            public override CultureInfo CurrentUICulture
+            {
+                get { return CultureInfo.CurrentUICulture; }
+            }
+            public override Guid InstanceId
+            {
+                get { return _instanceId; }
+            }
+            public override string Name
+            {
+                get { return "DummyTestHost"; }
+            }
+            public override PSHostUserInterface UI
+            {
+                get { return null; }
+            }
+            public override Version Version
+            {
+                get { return new Version(1, 0); }
+            }
+            public override void EnterNestedPrompt() { }
+            public override void ExitNestedPrompt() { }
+            public override void NotifyBeginApplication() { }
+            public override void NotifyEndApplication() { }
+            public override void SetShouldExit(int exitCode) { }
+            public void PushRunspace(Runspace runspace) { }
+            public void PopRunspace() { }
+            public bool IsRunspacePushed { get { return false; } }
+            public Runspace Runspace { get { return _runspace; } private set { _runspace = value; } }
```

Comment:
The test host Push/Pop methods are not needed for the test.  Only the Runspace property is needed to provide access to the host runspace.